networks/ip.go

```
@@ -0,0 +1,6 @@
+package networks
+
+
+type IPAddress struct {
+ value string,
```

This should raise a golangci error, making the action to fail, will remove once we assert that the CI fails

---

i would rather remove this for this PR and push it up as a separate one that executes github actions